lib/PAUSE.pm

```diff
@@ -345,7 +345,7 @@ sub owner_of_module {
                    FROM mods where modid = ?},
                  primeur => qq{SELECT package,
                           userid
-                   FROM primeur where LOWER(package) = LOWER(?)},
+                   FROM primeur where lc_package = LOWER(?)},
```

Comment:
Any reason you left LOWER(?) in this one place but changed all others to use `lc`?

Reply:
Or it might be better to repeat selectrow_arrayref without the for loop.

Reply:
Ah, I see. I think leaving LOWER(?) here is fine then